torch/csrc/distributed/rpc/tensorpipe_agent.h

```diff
@@ -11,12 +11,21 @@
 #include <c10d/Store.hpp>
 #include <torch/csrc/distributed/rpc/rpc_agent.h>
 
+#include <tensorpipe/tensorpipe.h>
```

Comment:
I think that for now we're trying to keep TensorPipe as a private dependency, which means we can't include its headers in PyTorch's private headers. I also don't think this is really needed, as we seem to only need it for `HAD_CUDA_IPC_CHANNEL`, and we can probably do without that.